home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/NewsApiTests/QueryNewsIT.java

```diff
@@ -0,0 +1,221 @@
+package com.softserveinc.ita.homeproject.api.tests.NewsApiTests;
+
+import com.softserveinc.ita.homeproject.ApiException;
+import com.softserveinc.ita.homeproject.api.NewsApi;
+import com.softserveinc.ita.homeproject.api.tests.query.NewsQuery;
+import com.softserveinc.ita.homeproject.api.tests.utils.ApiClientUtil;
+import com.softserveinc.ita.homeproject.api.tests.utils.QueryFilterUtil;
+import com.softserveinc.ita.homeproject.model.CreateNews;
+import com.softserveinc.ita.homeproject.model.ReadNews;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class QueryNewsIT {
+    private NewsApi newsApi;
+    private Integer pageNumber = 1;
+    private Integer pageSize = 10;
+
+    private final CreateNews expectedNews = new CreateNews()
+            .description("description")
+            .source("source")
+            .text("text")
+            .photoUrl("photoUrl");
+
+    @BeforeEach
+    public void setUp() {
+        newsApi = new NewsApi(ApiClientUtil.getClient());
+        expectedNews.setTitle(RandomStringUtils.randomAlphabetic(7));
+    }
+
+    @Test
+    void getAllNewsByTitleTest() throws ApiException {
+        newsApi.addNews(expectedNews);
+
+        List<ReadNews> actualListNews = new NewsQuery
+                .Builder(pageNumber, pageSize, newsApi)
```

Comment:
I think pageSize and pageNumber should be as other query parameters. There is no need make it required.